x-pack/plugins/actions/server/saved_objects/actions_migrations.ts

```diff
@@ -219,6 +229,29 @@ const addUsesTableApiToServiceNowConnectors = (
   };
 };
 
+const addIsOAuthToServiceNowConnectors = (
+  doc: SavedObjectUnsanitizedDoc<RawAction>
+): SavedObjectUnsanitizedDoc<RawAction> => {
+  if (
+    doc.attributes.actionTypeId !== '.servicenow' &&
+    doc.attributes.actionTypeId !== '.servicenow-sir' &&
+    doc.attributes.actionTypeId !== '.servicenow-itom'
+  ) {
+    return doc;
```

Comment:
just a nit :)
Don't you already check this at the lines 84-86?
and maybe `[".servicenow",".servicenow-sir", ".servicenow-itom"].includes(doc.attributes.actionTypeId)`

Reply:
Good catch! I removed the duplicate check :)